### PR TITLE
Implements support to venmo account

### PR DESCRIPTION
**Pull request by andrewaguiar**
Adds support to Venmo account payment method

### CODE REVIEW

File: test/integration/payment_method_test.exs [hunk 1]
```diff
@@ -290,6 +290,76 @@ defmodule Braintree.Integration.PaymentMethodTest do
...
+
+    {:ok, venmo_account} =
+      PaymentMethod.create(%{
+        customer_id: customer.id,
+        payment_method_nonce: Nonces.venmo_account(),
+        options: %{make_default: true}
+      })
+
+    assert venmo_account.username == "venmojoe"
+    assert venmo_account.default == true
```
Comment (sorentwo) [inline test/integration/payment_method_test.exs:325]: No need for the double equals, assert covers truthiness.

```suggestion
    assert venmo_account.default
```

**Review by sorentwo — APPROVED** [review]
Thanks! This is nicely done.

**Comment by andrewaguiar** [comment]
@sorentwo done, converted to `assert venmo_account.default` on paypal method as well

**Comment by sorentwo** [comment]
Thanks!

**Pull request merged**